src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -192,12 +192,15 @@ private void generateStatusMessage(List<String> versionToQuery,
         }
 
         // the "Search More" button to continue searching from the previous fromDate
-        status.append("<button class=\"btn-link\" id=\"button_older\" onclick=\"submitFormAjax("
-                      + nextEndTimeToSearch
-                      + ");\">Search More</button><input id=\"ifShowAll\" type=\"hidden\" value=\""
-                      + data.getIfShowAll()
-                      + "\"/><input id=\"ifShowTestData\" type=\"hidden\" value=\""
-                      + data.getIfShowTestData() + "\"/>");
+        String searchMoreButton = "<button class=\"btn-link\" id=\"button_older\"/>"
+                + "Search More</button><input id=\"ifShowAll\" type=\"hidden\" value=\""
+                + data.getIfShowAll()
+                + "\"/><input id=\"ifShowTestData\" type=\"hidden\" value=\""
+                + data.getIfShowTestData() + "\"/>"
+                + "<input id=\"nextEndTimeToSearch\" type=\"hidden\" value\""
+                + nextEndTimeToSearch
+                + "\"/>";
```

Comment:
I added the info into a data-* attribute in that admin log activity test PR that I'll probably put on hold for now. Think that way's better since it doesn't add extra nodes to the DOM.